joomla/joomla-cms
# [4.1] Schedule Tasks notification dates

Change to use DATE_FORMAT_LC instead of RFC822 to enable translated dates in the notification emails.

## To test 
make sure that your test site will send emails
Create a scheduled task and set the notification for task success to enabled
Install russian language and set to default for admin
Go to the list of scheduled tasks and perform a test run of the scheduled task.

### Before PR
The email will look similar to
> Задача #3, checking images, успешно выполнена Fri, 15 Apr 2022 08:54:34 +0000.

### After PR
The email will look similar to
> Задача #3, checking images, успешно выполнена Пятница, 15 апреля 2022, 09:09.

Pull Request for Issue #36522 .
